.github/sync.yml

```diff
@@ -0,0 +1,6 @@
+fury-project/fury-sites@main:
+  - source: docs/guide/
```

Comment:
source docs are not in guide directory, I'll submit a PR to update it

Reply:
OK